### PR TITLE
format log messages

### PR DESCRIPTION
**Pull request by sethisernhagen**
- use str.format() for log messages

### CODE REVIEW

File: microcosm_pubsub/handlers.py [hunk 1]
```diff
@@ -64,7 +64,9 @@ def __call__(self, message):
 
     def on_call(self, message, uri):
         self.logger.debug(
-            "Starting {handler}",
+            "Starting {handler}".format(
```
Comment (cdm-ium) [inline microcosm_pubsub/handlers.py]: Using a string format here is redundant. The console logger will do the formatting for you and in loggly the format string will show up as is (so you can search by it) and the values in extra will be added to the Json.

**Comment by cdm-ium** [comment]
The way the formatters in `microcosm_logging` are intended to work are:

- There is a console logger which will format values from extra into the string specified in the log message
- There is a json logger which will *not* format values from extra into the log message instead it puts the values from extra in the json that is logged and maintains the format string as is so you can search by it.

**Comment by sethisernhagen** [comment]
@cdm-ium Thanks for the explanation!  This is what I'm seeing in diego logs right now -```json:
message: Handled {handler}
handler: Sendnewworkspaceemail
uri: https://juvenal.dev.globality.io:5403/api/v2/service_provider_match/92ab6066-d9d2-4688-9215-8d6e8dc9fcc0```

**Comment by cdm-ium** [comment]
@sethisernhagen That's expected for what should show up in loggly. The logs on the box should have things actually formatted in.

**Comment by sethisernhagen** [comment]
@cdm-ium Got it. Sorry for the confusion. Closing.

**Comment by cdm-ium** [comment]
One bonus feature that we could add is having both the format string and the formatted string show up in two separate fields in loggly for reading pleasure as well as searchability.

**Comment by jessemyers** [comment]
@cdm-ium please!

**Pull request closed without merging**